README.md

```diff
@@ -49,9 +49,13 @@ Http request to list mosquitto users.
 
 `curl --location --request GET 'localhost:8080/list'`
 
+Http request to get mosquitto user by ID (returned by add enpoint).
+
+`curl --location --request GET 'localhost:8080/getById/5f32aa5de6e99db14a4a2b35'`
```

Comment:
probably all operations should have following urls: `localhost:8090/creds/<id>` and method PUT/GET/DELETE for update/obtain/remove single element, and `localhost:8090/creds` and method POST/GET for creation new/obtaining whole list.